mcan/src/config.rs

```diff
@@ -191,6 +191,13 @@ pub enum Mode {
     /// data) is enabled. This does not prevent use of classic CAN frames.
     Fd {
         /// If `true`, FD frames can be transmitted with bit rate switching.
+        ///
+        /// Regardless of this setting, data phase timing still must be
+        /// configured as *reception* of bit-rate-switched messages is still
+        /// possible
+        ///
+        /// This setting must be `true` if construction of bit-rate-switched
+        /// message is to succeed.
```

Comment:
As I remember it, messages can still be constructed, but `transmit` will return an error. Is that not the case?

The final paragraph seems redundant with the first line of the doc comment. If we want to emphasize the need to set it, maybe we could do something like this:

```suggestion
        /// If `true`, FD frames can be transmitted with bit rate switching.
        /// Otherwise, attempts to transmit FD frames will return errors.
        ///
        /// Regardless of this setting, data phase timing still must be
        /// configured as *reception* of bit-rate-switched messages is still
        /// possible.
```

What do you think?